src/ui/server/controllers/insurance/business/map-and-save/company-details/index.ts

```diff
@@ -51,15 +50,7 @@ const companyDetails = async (formBody: RequestBody, application: Application, v
       } = application;
 
       if (shouldNullifyCompanyDifferentAddress(hasDifferentTradingAddress, fullAddress)) {
-        console.info('Mapping and saving application - business - company details - nullifying different trading address data');
-
-        const nullifiedData = nullifyCompanyDifferentTradingAddress();
-
-        saveResponse = await saveAddress.companyDifferentTradingAddress(application, nullifiedData);
-
-        if (!saveResponse) {
-          return false;
-        }
+        await nullify.companyDifferentTradingAddress(application);
```

Comment:
should we be checking the return value of this incase it's false so we can return false? as in `const response = await nullify.companyDifferentTradingAddress(application);` if !response return false

Reply:
Good shout - updated, also updated for "Loss payee".